usabilla-react-native/README.md

```diff
@@ -40,13 +43,42 @@ react-native link usabilla-react-native
 To use this Bridge, please make sure you are using XCode 9.1 or above.
 
 ## Features
-The first version of Usabilla for React Native `0.1.0` allows you to :
-- Load and show a Passive Feedback form.
-- Submit the results of the form.
+### Load a Passive Feedback form
+In order to load a Passive Feedback form with the Usabilla library you need to call:
+
+`usabilla.loadFeedbackForm("YOUR_FORM_ID_HERE")`
+
+This method only performs the call to fetch the Passive Feedback form and in order to handle its response there are two methods available:
+
+- `usabilla.setFormDidLoadSuccessfully(usabilla.showLoadedForm())`: Sets the callback to execute when the form is fetched successfully
+- `usabilla.setFormDidFailLoading((error) => console.log(error))`: Sets the callback to execute when the form fails to be fetched. This callback has a parameter `error` containing a text explaining what happened
+
+**NOTE**: Usabilla provides a standard callback for showing a successfully fetched form with the method `usabilla.showLoadedForm()`
+
+**[iOS ONLY]**: When a user submits or closes the form, it is possible to get a callback containing information about the submission using the method:
+
+`usabilla.setFormDidClose((reminder) => console.log(reminder))`
+
+This callback has a parameter containing the information:
+  - formId (string)
+  - isRedirectToAppStoreEnabled (boolean)
+
+### Submit the results of the form
+This functionality is embedded in the native Usabilla library and there is no need to perform any specific action from the React Native environment.
+
+### Support for custom variables
+
+In order to set custom variables in the Usabilla native library it's necessary to call the method:
+
+`usabilla.setCustomVariables(customVars)`
```

Comment:
I'm a bit concerned with naming here...

Since expression "custom variables" was a too vague and didn't mean much I got rephrased to "customer input context", which is already being used by the new Search API:

https://github.com/usabilla/search-api/blob/17f4b92c92d850846aac1563628dd805ea090ab3/docs/openapi/feedback/items.yaml#L49-L51

Which means that once you decide to rename the method we'll a BC breaking change in a project which was just released, so I'd suggest you to already adopt the new terminology to prevent this issue.

Reply:
I see and agree, although it would mean to change it here and also in other projects (both readme and code bases) for Android and iOS.
Nothing specific against that, but @gPinato reported how this matter has been extensively discussed and yet, no alignment has been found (or perhaps not understood or communicated) since both Web and Mobile are both still using the terminology "custom variables"